src/LuaTransformer.ts

```diff
@@ -177,6 +177,10 @@ export class LuaTransformer {
         return tstl.createDoStatement(this.transformStatements(block.statements), undefined, block);
     }
 
+    public tstlIdentifier(name: string): string {
```

Comment:
Having doubts if this should be a function. It would probably be better to just format the string inside the import declaration function, since that is the only place it's used anyway

Reply:
Ok, I moved the method into a local arrow function. I still like the function as it is more readable and maintainable imo.

Reply:
Was thinking more along the lines of `const importName = "__TSTL_" + path.basename(importPath)` but this is okay too

Reply:
Yeah that does make more sense when used only here. On the other hand wouldn't it be nice to have some common prefix for all tstl auxiliary variables?